lib/node_modules/@stdlib/blas/base/sdot/src/sdot_f.c

```diff
@@ -0,0 +1,42 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+ * Compute the dot product of `X` and `Y`
+ *
+ * @see <a href="http://www.netlib.org/lapack/expolore-html/df/d28/group__single__blas__level1.html">sdot</a>
+ */
+#include "sdot.h"
+#include "sdot_fortran.h"
+
+/**
+* Computes the dot product of two single precision floating point
+* vectors
```

Comment:
```suggestion
* vectors.
```

No need to precede `vectors` with a hard line break.